tests/e2e/projects/harmony_with_mbr/harmony_with_mbr.spec.js

```diff
@@ -87,6 +84,7 @@ test.describe('Harmony with MBR', () => {
 
           // Expect the URL to be updated with `ess=t`
           await expect(page).toHaveURL(/pg\[1\]\[ess\]=t/)
+          await page.waitForURL(/zoom=3/, { timeout: 3000 })
```

Comment:
Is this a different zoom level than before? If it is we need new screenshots for these tests

Reply:
I had to bring down the pixel diff to get the `--update-snapshots` to work those should now we updated